### ISSUE TITLE
Script for Golf Tee Times

### ISSUE THREAD

**Issue by mikeboxelder** [mikeboxelder/boxangeles#1]
Is there any way to have a script that would parse through the responses on [this golf booking website](https://cityoflapcp.ezlinksgolf.com/index.html#/search) and book tee times?

Ideally it would on 10 days out schedule the earliest tee time available for a Saturday at certain selected courses.

**Comment by derrickbeining** [comment]
@mikeboxelder how do you envision this script getting executed? Are you wanting something you can run on the command line? A native desktop app with a graphical UI? Or are you imagining something more like a cron job on some server that runs automatically on a schedule?

**Comment by mikeboxelder** [comment]
@derrickbeining Cron job -- all the times get swallowed up by bots when they get posted, so by the time I get there, there isn't much left. So ideally it runs automatically on a schedule.

**Comment by derrickbeining** [comment]
Interesting. Who's making these bot?!

A cron job would be best implemented on something like AWS Lambda, I think. That sound alright to you?

**Comment by derrickbeining** [comment]
@mikeboxelder also, do you have a preference on what programming language the script should be implemented in or would I be free to use whatever I like?

**Comment by mikeboxelder** [comment]
Unclear, but [the city aknowledged it was happening](https://www.reddit.com/r/golf/comments/iqde1p/ok_so_who_on_rgolf_has_been_using_a_bot_to_get_la/). I think if this just grabs me one per week (or I can choose the dates to sweep in advance, i.e. sweep for Saturday 1/22 and Friday 2/4, et al) I should be fine.

TBH, I am a bit of a newb, so I am not sure what Lambda is or whether that is the correct route to go or not. I also obviously have no preference on language too, though ideally I can make simple revisions i.e. to days to book, or times to avoid.

**Comment by derrickbeining** [comment]
In order to develop this, I think I'd need to either have my own account on the booking website or be able to use yours, since you have to log in to complete the booking. It doesn't seem possible for me to create an account, so are you okay with giving me your credentials to be able to access the rest of the website's booking flow?

**Comment by mikeboxelder** [comment]
Yeah, I can email it to you.

**Comment by HarisRabbani** [comment]
Was this bounty completed or is it still open?

**Comment by mikeboxelder** [comment]
It was not @HarisRabbani

**Comment by ReethuVinta** [comment]
Can I work on this? If yes, I have a plan on how to proceed that I would like to discuss with you and I also need a few clarifications about the requirements. Can I get your e-mail id for the same?

**Comment by mikeboxelder** [comment]
Yeah @ReethuVinta you can. And my email is mikeboxelder@gmail.com
